src/test/java/com/epam/tc/hw7/site/pages/MetalsAndColorsPage.java

```diff
@@ -0,0 +1,45 @@
+package com.epam.tc.hw7.site.pages;
+
+import com.epam.jdi.light.elements.complex.Checklist;
+import com.epam.jdi.light.elements.complex.Selector;
+import com.epam.jdi.light.elements.complex.WebList;
+import com.epam.jdi.light.elements.complex.dropdown.Dropdown;
+import com.epam.jdi.light.elements.composite.WebPage;
+import com.epam.jdi.light.elements.pageobjects.annotations.FindBy;
+import com.epam.jdi.light.elements.pageobjects.annotations.Title;
+import com.epam.jdi.light.elements.pageobjects.annotations.Url;
+import com.epam.jdi.light.elements.pageobjects.annotations.locators.Css;
+import com.epam.jdi.light.elements.pageobjects.annotations.locators.JDropdown;
+import com.epam.jdi.light.elements.pageobjects.annotations.locators.UI;
+import com.epam.jdi.light.ui.html.elements.common.Button;
+import com.epam.jdi.light.ui.html.elements.complex.MultiSelector;
+import com.epam.jdi.light.ui.html.elements.complex.RadioButtons;
+
+@Title("Metal and Colors")
+public class MetalsAndColorsPage extends WebPage {
```

Comment:
hierarchy of JDI's PO should be like this:
Site -> Page -> Form -> Elements

In the other words, you have a Site and Page, but where is Form class?

Reply:
Fixed.